src/components/Events/events.styles.js

```diff
@@ -1,3 +1,249 @@
+import ArrowLeft from 'assets/arrow-left.svg';
+import ArrowRight from 'assets/arrow-right.svg';
+import bottomBorder from 'assets/bottom-border.svg';
 import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
 
-export const Title = styled.h3``;
+export const MainContainer = styled.div`
+  background: ${colors.alabaster};
+  padding: 2.25rem 0.75rem;
+
+  ${mq.desktop`
+    padding: 2rem 1.75rem;
+  `}
+`;
+
+export const Title = styled.h3`
+  color: ${colors.mineShaft};
+  font-size: 1.3rem;
+  margin: 0 0 1.1rem 0;
+  text-align: center;
+
+  ${mq.desktop`
+    font-size: 2rem;
+    margin: 0 0 2rem 0;
+  `}
+`;
+
+export const FilterList = styled.ul`
+  display: flex;
+  flex-direction: row;
+  gap: 2.25rem;
+  justify-content: center;
+  list-style-type: none;
+  margin: 0 0 1.12rem 0;
+  padding: 0;
+  text-align: center;
+
+  ${mq.desktop`
+    margin: 0 0 2rem 0;
+  `}
+`;
+
+export const ClickableItem = styled.button`
+  background: none;
+  border: none;
+  color: ${(props) => (props.activeFilter ? colors.green : colors.mineShaft)};
+  font-family: Montserrat;
+  font-size: 1rem;
+  font-weight: 600;
+  opacity: ${(props) => (props.activeFilter ? '1' : '0.5')};
+  padding: 0 0 0.25rem 0;
+  position: relative;
+
+  :hover {
+    cursor: pointer;
+  }
+
+  :after {
+    content: ${(props) => (props.activeFilter ? `url(${bottomBorder})` : '')};
+    position: absolute;
+    right: 0;
+    left: 0;
+    top: 0.62rem;
+  }
```

Comment:
why not simply using border top
```suggestion
  ${({ isActive }) =>
    isActive &&
    `
      :after {
        bottom: 0;
        border-top: 2px solid ${colors.green};
        border-top-left-radius: 0.25rem;
        border-top-right-radius: 0.25rem;
        content: '';
        left: 0.125rem;
        position: absolute;
        right: 0.125rem;
      }
    `};
```